spotbugsTestCases/src/java/RefComparisons.java

```diff
@@ -0,0 +1,49 @@
+public class RefComparisons {
```

Comment:
Can you please add some examples with enums as well?

Reply:
added as part of [df5db48](https://github.com/spotbugs/spotbugs/pull/2988/commits/df5db48bf8190dfff44297d9b823c63b2606b199)